changelog.md

```diff
@@ -291,9 +320,9 @@ For administrators:
   Elastic search index by running the following commands in the backend folder:
 
   ```sh
-  tracimcli index-drop
-  tracimcli index-create
-  tracimcli index-populate
+  tracimcli search index-drop
+  tracimcli search index-create
+  tracimcli search index-populate
```

Comment:
It is not necessary to change old version because user with old version can't seen the change here, user with old version have changelog of old version ;)
, also because I not sure about this syntax on tracim 3.9.0